stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 11.1.2
+version: 11.1.3
```

Comment:
Should be 11.1.4 for now @flecno

Reply:
Ok, I have bumped the version and rebased to master